### ISSUE TITLE
Consider compressing inlined files

### ISSUE THREAD

**Issue by colinbrislawn** [pnnl/hundo#11]
Those .tsv OTU tables can get big! Here's a method for compressing base64 inlined files
https://gist.github.com/slowkow/f9a43baafbf44817bc99e8274c8b3276 

It introduces one dependency for on-the-fly decompression: [pako](https://github.com/nodeca/pako)

Super low priority. ☺️

**Comment by colinbrislawn** [comment]
Does the current version provide `gz` files or do decompression on the fly? I would prefer to use `zip` files as they are natively supported on mac and windows.

To my surprise, there is no program that deflates to the `zip` format in parallel. pigz and 7z can make zip files, but [only use one thread per file](https://superuser.com/questions/433945/multithreaded-support-in-7za).
```
7z a -tzip -mmt8 all-sequences.fasta.zip all-sequences.fasta
pigz --zip -p 8 all-sequences.fasta            # totally not multithreaded
```

7z can use the BZip2 method (instead of deflate) to make zip files, but windows machines [can't open](https://social.technet.microsoft.com/Forums/windows/en-US/16521c3e-9905-492e-861c-5da3a8ceaeff/zip-files-fail-to-extract-or-open-error-code-0x80004005?forum=w7itprogeneral) the resulting archives.

```
7z a -mm=BZip2 -mmt8 all-zipped.zip all-sequences.fasta
```

That works great, but does not solve the compatibility problem. 🤷‍♂️
